deploy/template.yaml

```diff
@@ -36,6 +36,12 @@ objects:
                   value: ${AMQ_QUEUE_NAME}
                 - name: ENVIRONMENT
                   value: production
+                - name: RHSSO_SERVER_URL
+                  value: ${RHSSO_SERVER_URL}
+                - name: RHSSO_USER
+                  value: ${RHSSO_USER}
+                - name: RHSSO_PWD
+                  value: ${RHSSO_PWD}
```

Comment:
let's create a secret and store the password in the secret and then we can reference the secret from here.